docs/administering/faq.md

```diff
@@ -517,60 +517,10 @@ that we provide. Pay special attention to the `RewriteRule` stanzas.
-limits](https://letsencrypt.org/docs/rate-limits/), at which point all other visitors to Sandstorm
-would be unable to use grains. This issue alone makes the problem intractable.
-
-There are some additional problems as well.
-
-- **Loss of defense-in-depth for cross-site-request forgery protection and more.** Let's Encrypt
-  shares all certificate hostnames to the Certificate Transparency logs, which means you lose one of
-  Sandstorm's defense-in-depth mechanisms. Read the doc on [why we use wildcards](wildcard.md) for
-  more information on defense-in-depth via wildcards.
-
-- **Loss of privacy for static publishing URLs.** When users generate [static publishing
-  content](../developing/web-publishing.md) within a grain, they typically expect it to be private
-  until they choose a well-known domain name for the content. Let's Encrypt's integration with
-  Certificate Transparency would make all static publishing URLs by default.
-
-Some subproblems seem to have solutions, but they do not add up to a full solution for now.
-
-- **Latency.** The most straightforward way to implement Let's Encrypt dynamic provisioning with
-  Sandstorm would be for each new domain to be provisioned at the time it is created within
-  Sandstorm. This would mean that a visitor to your Sandstorm server would perceive a >10 second
-  latency introduced by Let's Encrypt while Let's Encrypt validates the domain and provides a
-  certificate. This could be solved by pre-generating a large number of subdomain certificates, but
-  it's not feasible to pregenerate enough. Consider that a server with 20 users and 20 grains would
-  need at least 400 certificates, but Let's Encrypt permits 20 certificates per week at the time of
-  writing. Additionally, Sandstorm periodically cycles these hostnames out of use. Let's Encrypt's
-  support for SAN certificates would allow 2,000 hostnames per week, but if a grain suddenly became
-  popular, new visitors would overwhelm this rate limit.
-
-One approach is to use `*.example.sandcats.io` for wildcard hostnames, but use a `BASE_URL` of `example.com`
-where `example.com` is certified via Let's Encrypt. However, this would mean that `example.com` contains
-IFRAME elements pointing at `*.example.sandcats.io` and these IFRAMEs may not be able to set cookies within
-the subdomain due to the increasingly-common **third-party cookie-blocking** done by browsers such as Chrome
-and Firefox.
+**Yes !** Let's Encrypt [recently started issuing wildcard certificates] (https://certbot.eff.org/faq/#does-let-s-encrypt-issue-wildcard-certificates), which are needed for Sandstorm
+for all the reasons documented in [our documentation on wildcard DNS.](wildcard.md).
+
+To make it all work, you can follow the instructions in [our documentation on reverse proxy](reverse-proxy.md) to configure nginx and Sandstorm. Just generate a certificate and a private key for both your base url (`example.com`) and the wildcard host you created (`*.example.com`).
```

Comment:
@paulproteus would probably be sad you used the word "just" here.

IMHO, if you have a working strategy for using Let's Encrypt with Sandstorm, my preference would be to remove the FAQ entry entirely, and make a separate page with the step-by-step instructions for getting a Let's Encrypt wildcard set up and configuring Sandstorm correctly to work with it.

Reply:
Sure, I'll try to find the time to write it ;) But basically I just reproduced the steps from the reverse proxy page.